tests/Settings/ContainerTestPlugin/plugin.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "TestContainerPlugin",
-    "api": 7,
+    "api": "7.6.0",
```

Comment:
This is incorrect. The `"api"` field needs to be an integer.

The field is basically deprecated, replaced by `"supported_sdk_versions"`. But we can't remove them since otherwise Cura 3.6 and earlier would accept it as a plug-in with fitting API.

```suggestion
    "api": 7,
```

Reply:
Thanks! It's good to pick up tasks that don't always fall to me so now and then :-)